collect_app/src/main/java/org/odk/collect/android/preferences/screens/ProjectPreferencesFragment.kt

```diff
@@ -68,14 +69,26 @@ class ProjectPreferencesFragment :
         findPreference<Preference>(CHANGE_ADMIN_PASSWORD_PREFERENCE_KEY)!!.onPreferenceClickListener = this
         findPreference<Preference>(PROJECT_MANAGEMENT_PREFERENCE_KEY)!!.onPreferenceClickListener = this
         findPreference<Preference>(ACCESS_CONTROL_PREFERENCE_KEY)!!.onPreferenceClickListener = this
+
+        if (inFormEntry) {
+            findPreference<Preference>(PROJECT_MANAGEMENT_PREFERENCE_KEY)!!.also {
```

Comment:
You can avoid calling `findPreference<Preference>(PROJECT_MANAGEMENT_PREFERENCE_KEY)` twice here and a few lines above by combining those two cases:
```
findPreference<Preference>(PROJECT_MANAGEMENT_PREFERENCE_KEY)!!.also {
            it.onPreferenceClickListener = this
            if (inFormEntry) {
                it.isEnabled = false
                it.setSummary(org.odk.collect.strings.R.string.setting_not_available_during_form_entry)
            }
        }
```